### ISSUE TITLE
Feedback for Samsung page

### ISSUE THREAD

**Issue by rsngfrce** [urbandroid-team/dont-kill-my-app#296]
I have a Samsung Galaxy S10e (SM-G970U), Android 11, USA, Snapdragon, AT&T, and my battery settings do not match what you display, plus there is a much simpler method of setting the app battery settings than you display. 

For Android 11, you state that you must go to each app individually and then access the battery settings for that app. Actually, on my phone, I go to the main app setting as you show, then hit the three dots menu in the upper right. From there, I select special access. Promise that menu I select optimize battery usage. Here to then displays all the apps that are not optimized or you can change it to display all apps, and then switch those to not being optimized. Much simpler than accessing this through each app individually. 

For the Galaxy 10, my settings are not the same as yours.From the 'Battery and device care' setting, I select 'battery' as you show. At that menu, I do not have the 3 dot menu you show. On that page, I have 3 battery related options, a "power saving mode" switch, a "background usage limits" option and "more battery settings". From "background usage limits", I have the option to put unused apps to sleep, plus listing for sleeping apps, deep sleeping apps and never sleeping apps. From "more battery settings", I have an adaptive battery setting. 

Refer to the attached screeshots for examples. 

![Android 11 - 1](https://user-images.githubusercontent.com/20010203/108302208-5f9b3500-7158-11eb-83d1-33650f4d6b5a.png)
![Android 11 - 2](https://user-images.githubusercontent.com/20010203/108302209-6164f880-7158-11eb-9eb0-fbabebe5e58c.png)
![Android 11 - 3](https://user-images.githubusercontent.com/20010203/108302211-61fd8f00-7158-11eb-9ca8-d375f765bd30.png)
![Android 11 - 4](https://user-images.githubusercontent.com/20010203/108302213-632ebc00-7158-11eb-92a2-b151dc96bf25.png)
![Galaxy S10e 1](https://user-images.githubusercontent.com/20010203/108302216-63c75280-7158-11eb-9222-b40971b2d3cf.jpg)
![Galaxy S10e 2](https://user-images.githubusercontent.com/20010203/108302220-645fe900-7158-11eb-862b-6c530188fd51.png)
![Galaxy S10e 3](https://user-images.githubusercontent.com/20010203/108302223-64f87f80-7158-11eb-8d0f-0e0fe065064e.png)

**Comment by petrnalevka** [comment]
Hello, big thanks for sharing this.. in the first screenshot I guess you can scroll down to see a full list of your installed apps? The Your apps section.. and than you unfortunately need to enable this for every app which you need to behave in a standard Android way.. :(

**Comment by rsngfrce** [comment]
Yes, just to be clear, in the first screenshot, you can scroll down to see the complete list of installed apps. However, you need to get to the forth screenshot, then you can see the apps that are not optimized or all apps and then you do have to enable it for every app as desired.

**Comment by petrnalevka** [comment]
Hello, big thanks for sharing this I now see the difference.. would you please share the screens without annotations? Can we use them as a resource on https://dontkillmyapp.com/samsung?

**Comment by rsngfrce** [comment]
Hi. I have attached complete screenshots without annotations of all the battery optimization screens that I am aware of. The are named in setting name format. Thus includes screenshots I didn't include before. If you have any questions, let me know. 

You may use these as you see fit. You provide an important resource and I learned some more about my battery optimization settings in doing this. 

There are no screenshots 06-09, 14. 

![00 settings](https://user-images.githubusercontent.com/20010203/108610959-d28df100-738e-11eb-9f3a-49b4cafb8abb.png)


![01 settings - apps](https://user-images.githubusercontent.com/20010203/108610961-d588e180-738e-11eb-8e87-4b100006674e.png)


![02 settings - apps - kebab menu](https://user-images.githubusercontent.com/20010203/108610962-d752a500-738e-11eb-9bc0-06cfaa9218c3.png)


![03 settings - apps - kebab menu - special access](https://user-images.githubusercontent.com/20010203/108610963-d91c6880-738e-11eb-8c6d-62771744a953.png)


![04 settings - apps - kebab menu - special access - optimize battery usage (apps not optimized)](https://user-images.githubusercontent.com/20010203/108610964-da4d9580-738e-11eb-9695-c4ad34ed8b01.png)


![05 settings - apps - kebab menu - special access - optimize battery usage (all)](https://user-images.githubusercontent.com/20010203/108610965-dc175900-738e-11eb-87d3-8869184985f6.png)


![10 - settings - device care](https://user-images.githubusercontent.com/20010203/108610966-dde11c80-738e-11eb-8a08-7cb79d783358.png)


![11 settings - device care - kebab menu](https://user-images.githubusercontent.com/20010203/108610967-df124980-738e-11eb-976f-5c4f528e69cc.png)


![12 settings - device care - kebab menu - automation](https://user-images.githubusercontent.com/20010203/108610969-e0dc0d00-738e-11eb-8b73-0918879e0cb1.png)


![13 settings - device care - kebab menu - automation -auto optimize daily](https://user-images.githubusercontent.com/20010203/108610970-e20d3a00-738e-11eb-9b4c-0d74fcabc4e1.png)


![15 settings - device care - battery](https://user-images.githubusercontent.com/20010203/108610971-e3d6fd80-738e-11eb-90e1-03b1cc04fcc9.jpg)


![16 settings - device care - battery - background usage limits](https://user-images.githubusercontent.com/20010203/108610972-e5a0c100-738e-11eb-970b-a8b2b93cb42b.png) 


![17 settings - device care - battery - background usage limits - sleeping apps](https://user-images.githubusercontent.com/20010203/108610973-e76a8480-738e-11eb-8e97-3c492f9a45fc.png)


![18 settings - device care - battery - background usage limits - sleeping apps - kabob menu](https://user-images.githubusercontent.com/20010203/108610977-e89bb180-738e-11eb-9231-4b0e49b5b662.png)


![19 settings - device care - battery - background usage limits - deep sleeping apps](https://user-images.githubusercontent.com/20010203/108610979-e9ccde80-738e-11eb-904b-299a1a06e4c8.png)


![20 settings - device care - battery - background usage limits -  deep sleeping apps - kebab menu](https://user-images.githubusercontent.com/20010203/108610980-ec2f3880-738e-11eb-8fd9-373d63d25b42.png)


![21 settings - device care - battery - background usage limits - never sleeping apps](https://user-images.githubusercontent.com/20010203/108610981-ee919280-738e-11eb-9e4d-4288be9d4b6c.png)


![22 settings - device care - battery - background usage limits - never sleeping apps - plus menu](https://user-images.githubusercontent.com/20010203/108610984-f0f3ec80-738e-11eb-9fe8-dab5208f889c.png)


![23 settings - device care - battery -more battery settings](https://user-images.githubusercontent.com/20010203/108610987-f3564680-738e-11eb-821d-9945c0d8dfae.png)

**Comment by rsngfrce** [comment]
Actually, I have recently allowed the Don't Kill My App app to run uninterrupted on my Samsung Galaxy S10e (SM-G970U) on Android 11 several times and I have always gotten 100's in all three categories, causing me to wonder why Samsung is rated so poorly by Don't Kill My App. These 100's are despite the fact that since upgrading to Android 11, I am having TERRIBLE issues with background apps being killed, which is due to memory issues I believe, rather than battery optimization issues.
